docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +337,96 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Experiencing latency in leaf cluster Nodes?">
+
+### Proxy Jump
+
+In the generated OpenSSH client configuration, the `ProxyCommand` for each leaf 
+cluster connects through the root cluster's Proxy Service. In scenarios where
+leaf cluster Proxy Services are reachable by SSH client, you might prefer to
+connect directly through the leaf proxies to achieve lower latency.
+
+To achieve this, update the `ProxyCommand` of the leaf cluster's configuration
+block to use the leaf Proxy Service as a jumphost, using the `-J` flag.
+
+```txt
+Host *.{{ .NodeName }}.leaf1.example.com
+   Port 3022
+   ProxyCommand tsh proxy ssh -J proxy.leaf1.example.com:443 %r@%h:%p
+```
+
+### Proxy Templates
+
+To streamline this customization, you can configure Proxy Templates, which
```

Comment:
"this customization" can be more specific. Maybe replace this paragraph with:

````markdown
With Proxy Templates, `tsh` will dynamically determine the address of the
Proxy Service to connect to based on the address of the destination
host in your `ssh` command.
````